### DISCUSSION TITLE
Ring resonator with a Lorenztian dispersion model returns no frequencies

### DISCUSSION THREAD

**Discussion by rerdo** [NanoComp/meep#2030] [Q&A] [answered]
Dear all,

I am trying to find resonance frequencies for a ring resonator made of Lithium Niobate on top of silicon dioxide, covered at the top with a polymer with a refraction index equal to 1.37. I am using cylindrical coordinates. To simulate Lithium niobate and silicon dioxide, I am using the Lorentzian model from meep.materials LiNbO3 and fused_quartz. The simulation seems to work fine at a lower resolution < 80. However, harminv is no longer returning any frequencies when increasing the resolution to 100. There is no error, but it returns nothing. I saw the FAQ questions and tried to reduce the courant factor but could not solve it. Any ideas? 

Best,

Renato

**Comment by smartalecH** [comment]
Maybe try posting some code. And highlight the change which appears to "break" harminv.

**Comment by stevengj** [comment, marked as answer]
Try plotting a few snapshots of the fields (or a movie) to see what is happening.

**Reply by stevengj** [reply]
(You could also try nondispersive materials based on the values for your bandwidth of interest.   The materials library is often overkill anyway because it wastes a lot of computational effort trying to match the dispersive response over a wide bandwidth, when often what you need is only a narrow bandwidth.)
